File: Sum Kind of Problem/sumkindofproblem.js
var n = parseInt(readline());
for (let i = 1; i <= n; i++) {
    let nums = readline().split(" ");
    let k = nums[0];
    let p = parseInt(nums[1]);
    console.log(k+" "+((p*(p+1))/2 >> 0)+" "+(p*p)+" "+(p*(p+1)));
}

File: Sum Kind of Problem/SumKindOfProblem.java
import java.io.*;

public class SumKindOfProblem {
    public static void main(String[] args) throws IOException {
        InputStreamReader inp = new InputStreamReader(System.in);
        BufferedReader sc = new BufferedReader(inp);
        PrintWriter writer = new PrintWriter(System.out);
        int n = Integer.parseInt(sc.readLine());
        for (int i = 0; i < n; i++) {
            String[] nums = sc.readLine().split(" ");
            int k = Integer.parseInt(nums[0]);
            int p = Integer.parseInt(nums[1]);
            writer.println(k+" "+(p*(p+1))/2+" "+p*p+" "+p*(p+1));
        }
        writer.flush();
    }
}